spec/services/redirection_creation_spec.rb

```diff
@@ -45,4 +45,17 @@
       expect(result).to be_nil
     end
   end
+
+  context "when the site is closed" do
```

Comment:
```suggestion
  context "when the 'ring is closed" do
```

Super small point, but I liked reading `'ring` in the spec above. It feels confidently flippant.

Reply:
confidently flippant is extremely on brand for you

Reply:
❤️